implementations/rust/ockam/ockam_node/src/context.rs

```diff
@@ -99,6 +115,7 @@ impl Context {
                 sender,
                 address,
                 mailbox,
+                async_drop_sender: None,
```

Comment:
Same comment: not obvious that we only create worker's Context here, so that drop handle is None